change_log/next/FE-2544-TranslationBugInPagerComponent.yml

```diff
@@ -0,0 +1 @@
+Bug Fixes: "Fixes translation Bug in Pager Component (Show was not translateable) (Components: Pager)."
```

Comment:
This can be removed as we no longer need change log files as we have adopted semantic releasing